src/java.base/share/classes/java/util/concurrent/ForkJoinTask.java

```diff
@@ -1857,4 +1874,39 @@ final void onRootCompletion() {
         public final void setRawResult(Void v) { }
         final Object adaptee() { return callable; }
     }
+
+    /**
+     * Adapter for Callable-based interruptible tasks with timeout actions.
+     */
+    @SuppressWarnings("serial") // Conditionally serializable
```

Comment:
Conditioned on what? 🤔

Reply:
Apparently this is meaningful to someone who added this in a java.util.* serialization check pass.